app/services/import_polygons.rb

```diff
@@ -1,6 +1,6 @@
 class ImportPolygons
   BUFFER_API_URL = "https://tasks.arcgisonline.com/arcgis/rest/services/Geometry/GeometryServer/buffer?".freeze
-  BUFFER_DISTANCES_IN_MILES = [5, 10, 15, 20, 25].freeze
+  BUFFER_DISTANCES_IN_MILES = Search::RadiusSuggestionsBuilder::RADIUS_OPTIONS
```

Comment:
Nice DRYing. Can we just use the RADIUS_OPTIONS directly instead of assigning it to another constant?